framework/branch-key-store.md

```diff
@@ -652,7 +652,7 @@ This operation MUST call AWS DDB `GetItem`.
 Every attribute except `enc` on the AWS DDB response item MUST be converted to a set of key value pairs (a Map)
 which is the [branch key context](#branch-key-context).
 Every key in the constructed branch key context except tableName MUST exist as a string attribute in the AWS DDB response item.
-The [encryption context](#encryption-context) MUST be built from branch key context by extracting keys with prefix `aws-crypto-ec:` and then dropping the prefix.
+The [AWS KMS Encryption Context](#aws-kms-encryption-context) MUST be built from branch key context by extracting keys with prefix `aws-crypto-ec:` and then dropping the prefix.
```

Comment:
I think you want a very similar statement that describes how to create the branch key context on Create.